src/ostree/ot-builtin-prune.c

```diff
@@ -343,5 +377,8 @@ ostree_builtin_prune (int argc, char **argv, OstreeCommandInvocation *invocation
     g_print ("Deleted %u objects, %s freed\n",
              n_objects_pruned, formatted_freed_size);
 
+  if (opt_commit_only) {
+    g_print("NOTE: this only includes commit objects\n");
+  }
```

Comment:
Mmmm...a bit of a bikeshed but I don't think we need this print.  The caller explicitly asked for this via `--commit-only`, it's a bit like ordering a sandwich and then the person giving it to you says "this is a sandwich" :wink:

Reply:
Personally, I felt a little confused when the `Total objects: ` value printed out didn't match the total number of objects in the repo. However, I realised that this only includes the number of objects traversed or considered for pruning. I thought that message might clear things up, but maybe it doesn't achieve what I was going for.

Maybe a compromise would be to change `Total objects:` to `Total objects traversed:`?

Reply:
Thats looks great!